header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -407,16 +409,33 @@ public function customize_register( WP_Customize_Manager $wp_customize ) {
 			? $this->description
 			: '';
 
-		$wp_customize->add_panel(
-			$this->panel,
-			array(
-				'priority'       => 25,
-				'capability'     => 'edit_theme_options',
-				'theme_supports' => Settings\Config::get_support(),
-				'title'          => $title,
-				'description'    => $description,
-			)
-		);
+		if ( class_exists( 'Neve_Pro\Modules\Header_Footer_Grid\Customizer\Custom_Panel' ) ) {
```

Comment:
I would suggest using an action like `hfg_after_builder_registered` which will get the `$wp_customizer` object. Later on we can hook in Neve pro to this action, remove the panel and add our new one, rather than using this if/else logic

What do you think?

Reply:
Much better.